scripts/repl.js

```diff
@@ -85,20 +85,24 @@ var cmi_files = [
   `js_float`,
   `js_json`,
 
-  `arrayLabels`,
-  `bytesLabels`,
-  `complex`,
-  `gc`,
-  `genlex`,
-  `listLabels`,
-  `moreLabels`,
-  `queue`,
-  `scanf`,
-  `sort`,
-  `stack`,
-  `stdLabels`,
-  `stream`,
-  `stringLabels`,
+  /*
+  These files cause troubles when compiled with JSOO (v3.4.0)
+  Be aware, if those are included you will get an error stating something like "/static/cmis/scanf.cmi : file already exists"
+  */
```

Comment:
why is the case?

Reply:
cc @jchavarri

Reply:
@bobzhang I am not sure. I have searched through git history to see if something changed in BuckleScript recently but I can't find anything relevant. And as far as I know, jsoo does not include cmis if the `--toplevel` flag is there (cc @hhugo to confirm).

@ryyppy Do you have any ideas why this could be?

Reply:
@jchavarri, jsoo **does** include cmis if --toplevel is passed. You can disable that behavior with `--disable does`

Reply:
Just tried uncommenting our cmi files and added `--disable withcmi` and could verify that it makes jsoo compile, but the resulting playground bundle will fail with runtime errors when you try to compile some code in the browser (probably missing some compiled deps with that setting). I guess we should just keep it the way it is, since it already works fine.